### ISSUE TITLE
Photo Shared through Twitter Button didn't Tweet

### ISSUE THREAD

**Issue by misslivirose** [mozilla/hubs#1838]
**Description**
I took a snapshot in a Discord-authed room and used the 'Tweet' button to try to tweet it. While it appeared to go through on the Hubs side, the photo and composed tweet didn't show up on Twitter.

**To Reproduce**
1. Take a photo with the camera (perhaps limited to a Discord bound room)
2. Click the 'Tweet' button
3. Compose and send Tweet

**Expected behavior**
The tweet appears

**Hardware** 
 - Device: Desktop
 - OS: Windows
 - Browser: Firefox

**Additional context**
This might be a usability issue. Specifically, when I first tweeted from Hubs, I was on a different computer and linked the 'ByHubs' twitter account instead of my own personal account. I could see there being something weird about how I'm authenticated.

**Comment by misslivirose** [comment]
P2 until a confirmed repro

**Comment by gfodor** [comment]
I see an error in sentry that implies that the POST to the twitter API failed for some reason. It seems likely perhaps the token was revoked/expired/etc. (Right now the twitter code is pretty dumb and doesn't handle invalid tokens properly)

**Comment by misslivirose** [comment]
I probably revoked the Twitter permission on the account. We should re-request auth in the case that it's failing

**Comment by gfodor** [comment]
Yep I agree, we should add that as a TODO -- the current twitter auth flow
is arguably half finished without dealing with revocations

**Comment by blairmacintyre** [comment]
twitter still seems broken in hubs cloud
